example/llm/get_openai_response.py

```diff
@@ -1,21 +1,26 @@
 import os
 from dotenv import load_dotenv
+
 load_dotenv()
```

Comment:
any reason why `load_dotenv` is above the rest of the imports? this seems odd to me

Reply:
I probably copied the code from somewhere and it was done like that... I feel like it is a bit weird too, but it makes sense as it is kind of a initial configuration

Reply:
I disagree, I think we should always import everything before doing any logic 👍